### PR TITLE
Edit L5 ET Synonyms

### LINKED ISSUE #1147: [Synonym] L5 ET Synonyms
**CL term**
L5 ET

**Synonym to be added**
**L5b neurons**
Reference DOI: 10.1016/j.neuron.2011.07.029
broad
______

**Thick-tufted layer 5 pyramidal neurons**
Reference DOI: 10.3389/fncel.2015.00233
broad
_____

**Burst-firing layer 5 neurons**
Reference DOI: https://doi.org/10.3389/fncel.2011.00001
broad
______

**Pyramidal tract (PT) neurons**
Reference DOI: 10.1038/nrn3469
narrow/exact
______

**Subcerebral projection neurons (SCPN)**
Reference DOI: https://doi.org/10.1016/j.neuron.2005.08.030
broad
______

**Pyramidal tract-like (PT- l)**
Reference DOI: https://doi.org/10.1016/j.cell.2019.09.023
related

_____

**Thick-tufted layer 5 neuron (TTL5)**
Reference DOI: 10.3389/fncel.2015.00233
broad

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #1147

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27978,11 +27981,12 @@ SubClassOf(obo:CL_4023040 obo:CL_4023008)
...
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "subcerebral projection (SCPN) neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "thick-tufted layer 5 pyramidal neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2011.07.029") oboInOwl:hasExactSynonym obo:CL_4023041 "L5b neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.cell.2019.09.023") Annotation(rdfs:comment "pyramidal tract-like is used because, unlike pyramidal tract neurons in the motor cortex, neurons in the structures like the auditory cortex do not project to the spinal cord.") oboInOwl:hasExactSynonym obo:CL_4023041 "Pyramidal tract-like (PT-l)")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fncel.2011.00001") oboInOwl:hasExactSynonym obo:CL_4023041 "burst-firing layer 5 neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1038/nrn3469") oboInOwl:hasExactSynonym obo:CL_4023041 "pyramidal tract (PT) neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2005.08.030") oboInOwl:hasExactSynonym obo:CL_4023041 "subcerebral projection (SCPN) neuron")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27988]: I think this one should be a broad synonym:
image

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27978,11 +27981,12 @@ SubClassOf(obo:CL_4023040 obo:CL_4023008)
...
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "burst-firing layer 5 neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "pyramidal tract (PT) neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "subcerebral projection (SCPN) neuron")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023041 "thick-tufted layer 5 pyramidal neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2011.07.029") oboInOwl:hasExactSynonym obo:CL_4023041 "L5b neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.cell.2019.09.023") Annotation(rdfs:comment "pyramidal tract-like is used because, unlike pyramidal tract neurons in the motor cortex, neurons in the structures like the auditory cortex do not project to the spinal cord.") oboInOwl:hasExactSynonym obo:CL_4023041 "Pyramidal tract-like (PT-l)")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fncel.2011.00001") oboInOwl:hasExactSynonym obo:CL_4023041 "burst-firing layer 5 neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1038/nrn3469") oboInOwl:hasExactSynonym obo:CL_4023041 "pyramidal tract (PT) neuron")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27987]: Given comment on pyramidal-tract like - this should probably be a narrow synonym.

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments

**Comment by shawntanzk** [comment]
Made the changes, also changed L5b neuron to broad synonym as L5b is defined by PT neurons instead of L5b neuron being PT neurons I think

**Review by patrick-lloyd-ray — APPROVED** [review]

**Pull request merged**